### ISSUE TITLE
508-defect-2 [COGNITION]: Supporting evidence page SHOULD be split into two pages to reduce cognitive load

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#19026]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Josh_

## User Story or Problem Statement

As a user with cognitive considerations, I expect:
- critical information to be announced to me in plain language
- clear instructions along with relevant examples to help me understand what I must upload
- focus to _not_ be lost on the page

## Details

The supporting evidence page may require more discussion around the following topics:
- Instructions to the user (has this been tested with Veterans? Do they understand what service treatment records are? Do they know what pages to upload, what is most important, etc?
- Do we need to have an upload to begin with? Can users enter the bare amount of information necessary to confirm manually to prevent mishaps related to uploads? [Gov.uk recommends to "Avoid uploading, if possible" for good reason](https://designnotes.blog.gov.uk/2017/02/14/some-design-tips-for-uploading-things/).
- Focus management gets lost in upload
- And more possibly

## Acceptance Criteria

- [ ] TBD

## Proposed Solution (if known)

This will require a meeting to resolve given its complexity. Tentatively, we may want to consider splitting this out into two separate pages following recent design system team guidance.

## Screenshots or Trace Logs

<img width="762" alt="Screen Shot 2021-01-26 at 9 55 26 AM" src="https://user-images.githubusercontent.com/14154792/105863457-eaa46780-5fbe-11eb-96c1-b428bd3d9bf5.png">
<img width="1792" alt="Screen Shot 2021-01-26 at 10 08 18 AM" src="https://user-images.githubusercontent.com/14154792/105863464-ebd59480-5fbe-11eb-9d92-637d5ded31be.png">
https://user-images.githubusercontent.com/14154792/105864168-ac5b7800-5fbf-11eb-9c44-ecb10c15ac70.mov
<img width="739" alt="Screen Shot 2021-01-26 at 10 19 55 AM" src="https://user-images.githubusercontent.com/14154792/105864675-2e4ba100-5fc0-11eb-9334-f1aa6cfff0b7.png">

**Comment by joshkimux** [comment]
More information in this slack thread:
https://dsva.slack.com/archives/C5AGLBNRK/p1611770008028900

**Comment by kevinstachura** [comment]
Spoke with SMEs at our weekly BDD meeting to discuss this item. Some key takeaways:

**1. We have no specific timetable for DoD to auto-provide Service Treatment Records.** If this were to happen near-term (next month or so), then we could probably just wait for that to occur and eliminate the need for SMs to upload STRs (and remove the associated screens) altogether. It seems that this change from DoD is likely several months out.

**2. It is useful for the MSCs to have the service member self-identify their documents as either 'medical' or 'dental'.** This is because MSCs must bucket these items themselves and it's something that assists them in their overall process. Failure to do so may slow down the process. 

**3. Using terms like 'photocopy' and 'scan' are accurate in describing the process some SMs undertake.** The SM will copy this information either via a CD, flash/thumb drive or some other digital means or may scan if they receive via paper. 

Other interesting notes: 

- The dental records are not part of the service treatment records. Rather, that is its own document and would need to be provided in addition to the service treatment records. This gives us an opportunity to differentiate and instruct service members on how to obtain this document, which is necessary for any dental-related condition they may be claiming.
